### PR TITLE
🙅🏼‍♂️ Create menu sidebar for modifying categories

### PR DESCRIPTION
**Pull request by ayanmallik**
## Description

- Create the sidebar for the "Manage Resources" tab
- Populate the sidebar with categories/subcategories from the get endpoint
- Style the X icon to appear as red

Fixes #<number>

## Screenshots

image

### CODE REVIEW

File: client/src/components/ResourceManager.jsx
Comment (jackieo5023) [inline client/src/components/ResourceManager.jsx:31]: What's the exapandIcon for?

File: client/src/components/ResourceManager.jsx
Comment (jackieo5023) [inline client/src/components/ResourceManager.jsx:41]: Most of the time we shouldn't need a div to wrap - we can just use a fragment (so `<>` and `</>`).  Lmk if we need a div for this case though

**Review by jackieo5023 — APPROVED** [review]
Looks great! A few things:
* This is kind of specific, but is it possible to move the arrow a bit up using some CSS? If it's too difficult don't worry about it though :) 
* Can we add the functionality of the down arrow changing to an up arrow if the menu is opened?
* I think we also want the edit and delete icons to be on the category menu as well
* We also want to be able to see the "Add subcategory" and "Add category" buttons (menu items?) as per the [design](https://www.figma.com/file/JDI6BvoiJBG5JtGHisQOMy/YMCA-NAWC-Sp21?node-id=0%3A1)

Don't worry about adding these points to this PR - feel free to merge this after getting another approval! You can split the above bullets into separate PRs to keep it more modular, I'm just mentioning them here to keep them on your mind

File: client/src/components/ResourceManager.jsx
Comment (ayanmallik) [reply client/src/components/ResourceManager.jsx:31]: The expandIcon (located on the right) is traditionally a dropdown arrow, although we wanted to disable that righthand arrow to place the edit/close icons in that spot. I made that happen by passing in an empty div.

**Review by nkonjeti — APPROVED** [review]
Looks good!🤩

**Pull request merged**
